src/lib.rs

```diff
@@ -296,3 +317,22 @@ pub async fn run(config: Config) -> Result<()> {
         }
     }
 }
+
+pub async fn image_present_locally(images: Images, search_for_image: String) -> bool {
+    match images.list(&Default::default()).await {
+        Ok(images) => {
+            for image in images {
+                for repo_tag in image.repo_tags {
+                    for tag in repo_tag {
+                        print!("{} | {}", tag, search_for_image);
```

Comment:
I'm not sure we should print() here -- let's just take it out?

Reply:
Ah, that was debugging. My bad there.